src/ZODB/serialize.py

```diff
@@ -699,7 +699,7 @@ def get_refs(a_pickle):
         if isinstance(reference, tuple):
             oid, klass = reference
         elif isinstance(reference, (bytes, str)):
-            data, klass = reference, None
+            oid, klass = reference, None
```

Comment:
Split in 2 lines to avoid the tuple packing/unpacking.

Reply:
But this way it's nicely symmetric with the case above, where reference is a tuple.